library/Booster/JsonRpc.hs

```diff
@@ -282,10 +289,10 @@ respond stateVar =
                         (Left ApplyEquations.SideConditionFalse{}, _) -> do
                             let tSort = externaliseSort $ sortOfPattern pat
                             pure $ Right (addHeader $ KoreJson.KJBottom tSort, [])
-                        (Left (ApplyEquations.EquationLoop terms), _) ->
-                            pure . Left . RpcError.backendError RpcError.Aborted $ map externaliseTerm terms -- FIXME
+                        (Left (ApplyEquations.EquationLoop _terms), _) ->
+                            pure . Left . RpcError.backendError $ RpcError.Aborted "equation loop detected" -- FIXME
```

Comment:
:+1: Good to be more specific in the error message (that was what the `FIXME` meant to refer to), and OK to not include the terms involved in the loop. We should just make sure they are logged, at least at `ErrorDetails` level.
```suggestion
                        (Left (ApplyEquations.EquationLoop _terms), _) ->
                            pure . Left . RpcError.backendError $ RpcError.Aborted "equation loop detected"
```